<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GensArchiveCleaner</class>
 <widget class="QMainWindow" name="GensArchiveCleaner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>956</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>GensArchiveCleaner</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Directory:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="LE_Folder">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="PB_Open">
        <property name="text">
         <string>...</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Base Files</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="CustomTreeView" name="TV_Base">
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="indentation">
            <number>0</number>
           </property>
           <property name="itemsExpandable">
            <bool>false</bool>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <attribute name="headerMinimumSectionSize">
            <number>5</number>
           </attribute>
           <attribute name="headerStretchLastSection">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="L_BaseItem">
             <property name="text">
              <string>0 items,</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="L_BaseSelected">
             <property name="text">
              <string>0 selected,</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="L_BaseError">
             <property name="text">
              <string>0 errors</string>
             </property>
             <property name="textFormat">
              <enum>Qt::RichText</enum>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="PB_Copy">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Copy Selected Base Files and Resources to...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PB_DeleteBase">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Delete Selected</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="title">
         <string>Resources</string>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="CustomTreeView" name="TV_Resource">
           <property name="selectionMode">
            <enum>QAbstractItemView::ExtendedSelection</enum>
           </property>
           <property name="indentation">
            <number>0</number>
           </property>
           <property name="itemsExpandable">
            <bool>false</bool>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
           <attribute name="headerMinimumSectionSize">
            <number>5</number>
           </attribute>
           <attribute name="headerStretchLastSection">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="L_ResourceItem">
             <property name="text">
              <string>0 items,</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="L_ResourceSelected">
             <property name="text">
              <string>0 selected,</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="L_ResourceUnused">
             <property name="text">
              <string>0 potentially unused</string>
             </property>
             <property name="textFormat">
              <enum>Qt::RichText</enum>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="PB_Clean">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Clean All Unused Resources</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="PB_DeleteResource">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Delete Selected</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomTreeView</class>
   <extends>QTreeView</extends>
   <header location="global">customtreeview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
